app/javascript/controllers/dropdown_menu_controller.js

```diff
@@ -1,12 +1,9 @@
 import { Controller } from "@hotwired/stimulus";
 
-console.log("Hello again")
```

Comment:
I noticed that too after the merge 😄 Was gonna remove it in a new PR but I guess here is fine too